README.md

```diff
@@ -1,6 +1,6 @@
 # Tessel 2 Docs
 
-[![Code of Conduct](https://img.shields.io/badge/%E2%9D%A4-code%20of%20conduct-blue.svg?style=flat)](https://github.com/tessel/project/blob/master/CONDUCT.md)
+![Code of Conduct](https://img.shields.io/badge/❤-code of conduct-blue.svg?style=flat)(https://github.com/tessel/project/blob/master/CONDUCT.md)
```

Comment:
This doesn't appear to render correctly without the spaces encoded:

<img width="1143" alt="screen shot 2017-08-06 at 8 18 30 pm" src="https://user-images.githubusercontent.com/3051193/29008419-985bbafe-7ae4-11e7-92c3-c96b4f512c7a.png">